dev-lang/zig/zig-0.9.1.ebuild

```diff
@@ -56,6 +56,7 @@ src_configure() {
 	local mycmakeargs=(
 		-DZIG_USE_CCACHE=OFF
 		-DZIG_PREFER_CLANG_CPP_DYLIB=ON
+		-DZIG_SINGLE_THREADED="$(usex threads OFF ON)"
```

Comment:
Hmm this doesn't seem to work, at least `-threads` fails,
```
FAILED: zig1.o /var/tmp/portage/dev-lang/zig-0.9.1/work/zig-0.9.1/build/zig1.o
cd /var/tmp/portage/dev-lang/zig-0.9.1/work/zig-0.9.1 && /var/tmp/portage/dev-lang/zig-0.9.1/work/zig-0. 9.1/build/zig0 src/stage1.zig -target native -mcpu=baseline --name zig1 --zig-lib-dir /var/tmp/portage/  dev-lang/zig-0.9.1/work/zig-0.9.1/lib -femit-bin=/var/tmp/portage/dev-lang/zig-0.9.1/work/zig-0.9.1/     build/zig1.o -fcompiler-rt -OReleaseFast --strip -fsingle-threaded -lc --pkg-begin build_options /var/   tmp/portage/dev-lang/zig-0.9.1/work/zig-0.9.1/build/config.zig --pkg-end --pkg-begin compiler_rt /var/   tmp/portage/dev-lang/zig-0.9.1/work/zig-0.9.1/lib/std/special/compiler_rt.zig --pkg-end
Invalid argument: -fsingle-threaded
See `/var/tmp/portage/dev-lang/zig-0.9.1/work/zig-0.9.1/build/zig0 --help` for detailed usage information
ninja: build stopped: subcommand failed.
 * ERROR: dev-lang/zig-0.9.1::gentoo failed (compile phase):
```
(same with 9999 I guess)

Reply:
My bad, I will create patch